internal/config/config_file_test.go

```diff
@@ -150,20 +153,149 @@ func Test_parseConfigFile(t *testing.T) {
 
 func Test_ConfigDir(t *testing.T) {
 	tests := []struct {
-		envVar string
-		want   string
+		name        string
+		onlyWindows bool
+		env         map[string]string
+		output      string
 	}{
-		{"/tmp/gh", ".tmp.gh"},
-		{"", ".config.gh"},
+		{
+			name:   "no envVars",
+			output: ".config.gh",
+			env: map[string]string{
+				"AppData": "",
+			},
+		},
+		{
+			name: "GH_CONFIG_DIR specified",
+			env: map[string]string{
+				"GH_CONFIG_DIR": "/tmp/gh_config_dir",
+			},
+			output: ".tmp.gh_config_dir",
```

Comment:
It's not really clear that this is a regexp pattern and that `.` is meant to match either style of slashes. You can use  static strings and`filepath.FromSlash()` here too to normalize path comparison across platforms